examples/stable_diffusion_v2/ldm/modules/attention.py

```diff
@@ -158,32 +206,56 @@ def rearange_in(x):
         attn = self.softmax(sim)
         out = ops.matmul(attn, v)
 
-        def rearange_out(x):
-            # (b*h, n, d) -> (b, n, h*d)
-            h = self.heads
-            b, n, d = x.shape
-            b = b // h
+        return out
 
-            x = self.reshape(x, (b, h, n, d))
-            x = self.transpose(x, (0, 2, 1, 3))
-            x = self.reshape(x, (b, n, h * d))
-            return x
 
-        out = rearange_out(out)
-        return self.to_out(out)
+class FlashAttention(nn.Cell):
+    def __init__(self, head_num, head_dim):
+        super(FlashAttention, self).__init__()
+        self.head_num = head_num
+        self.scale = head_dim ** -0.25
+        self.flash_attention = get_flash_attention(tiling_stgy_name='xunfei')
```

Comment:
could you please move tiling_stgy_name to init's arg?

Reply:
This is an internal parameter, and users are not expected to modify it.